tutor/templates/build/openedx/Dockerfile

```diff
@@ -138,9 +141,10 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt update \
     && apt install -y gettext gfortran graphviz graphviz-dev libffi-dev libfreetype6-dev libgeos-dev libjpeg8-dev liblapack-dev libmysqlclient-dev libpng-dev libsqlite3-dev libxmlsec1-dev lynx mysql-client ntp pkg-config rdfind
 
+
 # From then on, run as unprivileged "app" user
 # Note that this must always be different from root (APP_USER_ID=0)
-ARG APP_USER_ID=1000
+ARG APP_USER_ID=1001
```

Comment:
This has pretty big implications. For one, the UID on the host is very often 1000, so changing the internal UID is likely to have more implications in terms of ownership, especially for mounted volumes.  Thus, I think we should avoid running as a different UID. Instead, can we try out this suggestion to delete the existing ubuntu user? https://askubuntu.com/a/1515958

Reply:
Sure. I initially thought of doing that but it looked weird. Some online suggestions were to use the next available uid which was 1001 in this case.

Reply:
I have updated the PR, please take another look.

Reply:
@regisb Hi, a reminder on this. Thanks